### DISCUSSION TITLE
Jukebox mode / continuous playback based on genre (or configureable)

### DISCUSSION THREAD

**Discussion by BredMan0** [audioling/audioling#11] [Ideas]
This topic has been brought up before: https://github.com/jeffvli/feishin/issues/7

**Comment by tcurdt** [comment]
Supporting jukebox mode would bring multi-room audio (via snapcast) in reach. That would be awesome.
This would be very high on my list of priorities.

**Comment by BredMan0** [comment]
Adding on to this, Symfonium recently released an update adding different continuous playback modes, such as artist based, genre based, or simply random based. There is also an option to queue entire albums instead of single tracks. It would be great if these options were implemented in Audioling as well. 

Refer to: https://support.symfonium.app/t/7042
